### ISSUE TITLE
Pretrained Model on DFDC dataset

### ISSUE THREAD

**Issue by winterfell2021** [xaCheng1996/VFD#5]
Hi there, can u provide checkpoints trained on DFDC dataset? thx

**Comment by xaCheng1996** [comment]
Our critical contribution is to train a powerful deepfake discriminator based only on the general audio-visual dataset. DFDC is a forged video dataset and is only used as a test set in our experiment.

If you want to get a checkpoint on DFDC, please train from scratch with your own data.

**Comment by xaCheng1996** [comment]
Our critical contribution is to train a powerful deepfake discriminator based only on the general audio-visual dataset. DFDC is a forged video dataset and is only used as a test set in our experiment.

If you want to get a checkpoint on DFDC, please train from scratch with your own data.
